decidim-core/spec/mailers/notification_digest_mailer_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 module Decidim
   describe NotificationsDigestMailer do
-    let(:organization) { create(:organization, name: "O'Connor") }
```

Comment:
These kinds of organizations names are to detect the apostrophe bug that happens usually. Let's leave it or maybe refactor it to a context (`when the organization has an apostrophe in its name`)

Reply:
I understand this, but here we do not have any assertion to test the organization name is actually displayed like it should. Therefore, i do not see it relevant in this place